layouts/index.html

```diff
@@ -616,10 +615,10 @@ <h2><i class="fa-regular fa-books"></i> RAPIDS News </h2>
     new Chart(ctx3, {
         type: 'bar',
         data: {
-            labels: ['Jaccard', 'Core Number', 'Louvain', 'PageRank', 'Katz'],
+            labels: ['WCC', 'Triangles', 'PageRank', 'Louvain', 'BC k=100'],
```

Comment:
I think you could try these intermediate-length names? Otherwise I'm okay with what you wrote, but maybe it'd be nice to have a note in the text someplace to explain `WCC = Weakly Connected Components`, `BC = Betweenness Centrality`.
```suggestion
            labels: [['Weak', 'Components'], 'Triangles', 'PageRank', 'Louvain', 'Betweenness'],
```

Reply:
ok, I'll try adding the note.  FWIW, using WCC [is common](https://neo4j.com/docs/graph-data-science/current/algorithms/wcc/) but BC may not be.  The "k=100" is significant for achieving this speedup, so I'd prefer not to drop that.

Reply:
Thanks for the suggestion.  I think the final commit here looks good and matches the other plots best.  It uses abbreviated names and updates the existing note to also describe the abbreviated names.